scripts/Makefile.pdb

```diff
@@ -51,16 +58,16 @@ $(PDB_HOME)/libUTSimTop.so: $(PDB_HOME)/picker.f
 		--lang python \
 		--tdir $(PDB_HOME)/pyxscore \
 		--fs $(PDB_HOME)/picker.f \
-		-V "--no-timing;--threads;128;+define+DIFFTEST;-I$(NOOP_HOME)/build/generated-src" \
-		-C "-fPIC -lz -I$(PICKER_INCLUDE) -L$(PDB_HOME)/pyxscore -ldifftest -lpython$(PYTHON_VERSION) $(SIM_LDFLAGS)" \
-		--autobuild=false
-	ln -s ../pydifftest/_difftest.so $(PDB_HOME)/pyxscore/libdifftest.so
-	$(MAKE) -C $(PDB_HOME)/pyxscore
+		-V "--output-split;20000;--no-timing;--threads;128;+define+DIFFTEST;-I$(NOOP_HOME)/build/generated-src" \
+		-C "-fPIC -lz -I$(PICKER_INCLUDE) -L$(PDB_HOME)/pydifftest -ldifftest -Wl,-rpath='$$ORIGIN../pydifftest' -lpython$(PYTHON_VERSION) $(SIM_LDFLAGS)" 
 
 pdb-run: check-deps
 	LD_PRELOAD="$(PDB_LD_PRELOAD)" \
 	PYTHONPATH="$(PDB_PYTHONPATH)" \
-	python3 scripts/pdb-run.py
+  	python3 scripts/pdb-run.py $(PDB_ARGS) $(RUN_ARGS)
+
+pdb-test: 
+	$(MAKE) pdb-run IMAGE_FILE=ready-to-run/copy_and_run.bin BATCH_ARGS="-l --debug-level info -c 10000"
```

Comment:
set default value for `PDB_ARGS`, `IMAGE_FILE`, and `BATCH_ARGS`, then we can use `make pdb-run` (w/o args) to replace the `pdb-test` target